doc/manual.md

```diff
@@ -4773,8 +4773,10 @@ Example:
       echo "overflow!"
     except ValueError, IOError:
       echo "catch multiple exceptions!"
-    except:
-      echo "Unknown exception!"
+    except CatchableError:
+      echo "Catchable exception!"
+    except Defect:
```

Comment:
Not sure we should show that frequently how to catch Defects, that is supposed to be an "advanced feature"
(same for other `except Defect` in this PR)

Maybe instead a section about catching defect, saying it won't work with `--panics:on`, and discouraging it generally?